<launch>

	<node name="laser_wanderer" type="laser_wanderer.py" pkg="ta_lab1" output="screen" >
		<param name="speed" type="double" value=".2" />
		<param name="min_delta" type="double" value="-0.34"/>
		<param name="max_delta" type="double" value="0.34" />
		<param name="delta_incr" type="double" value="0.1133333333333"/>
		<param name="dt" type="double" value="0.01" />
                <!--<param name="dt" type="double" value="0.01" />-->
		<param name="T" type="int" value="300" />
                <!--<param name="T" type="int" value="300" />-->
		<param name="compute_time" type="double" value="0.09" />
		<param name="laser_spread" type="double" value="0.314" />
		<param name="laser_offset" type="double" value="1.0" />
		<param name="real_car" type="bool" value="False" />
		
		<!-- RF Parameters TODO get from other launch file -->
		<param name="theta_min" type="double" value="-1.2217304764" />
               <param name="theta_max" type="double" value="1.2217304764" />
               <param name="theta_count" type="int" value="360" />
	</node>

</launch>
